src/azure-cli/azure/cli/command_modules/ams/operations/mru.py

```diff
@@ -9,7 +9,13 @@
 import requests
 from azure.cli.core.util import CLIError
 from azure.cli.command_modules.ams._completers import get_mru_type_completion_list
-
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core._profile import _ACCESS_TOKEN
+from azure.cli.core._profile import _REFRESH_TOKEN
+from azure.cli.core._profile import _SERVICE_PRINCIPAL
+from azure.cli.core._profile import _USER_ENTITY
+from azure.cli.core._profile import _USER_TYPE
+from azure.cli.core._profile import _USER
```

Comment:
These should just be combined:
```Python
from azure.cli.core._profile import (
_ACCESS_TOKEN, _REFRESH_TOKEN, ...)
```

Reply:
I'm using those profile constants in the results of the profile calls. If I just replicate the strings in my code, and profile changes a string, I would definitely break.